scipy/optimize/_linprog_highs.py

```diff
@@ -0,0 +1,207 @@
+            slack : 1D array
+                The (nominally positive) values of the slack variables,
+                ``b_ub - A_ub @ x``.
+            con : 1D array
+                The (nominally zero) residuals of the equality constraints,
+                ``b_eq - A_eq @ x``.
+            success : bool
+                ``True`` when the algorithm succeeds in finding an optimal
+                solution.
+            status : int
+                An integer representing the exit status of the algorithm.
+
+                ``0`` : Optimization terminated successfully.
+
+                ``1`` : Iteration limit reached.
+
+                ``2`` : Problem appears to be infeasible.
+
+                ``3`` : Problem appears to be unbounded.
+
+                ``4`` : Numerical difficulties encountered.
+
+            nit : int
+                The total number of iterations performed in all phases.
+            message : str
+                A string descriptor of the exit status of the algorithm.
+
+    """
+
+    _check_unknown_options(unknown_options)
+
+    statuses = {
+        12: (1, "Iteration limit reached."),
+        9: (0, "Optimization terminated successfully."),
+        8: (3, "The problem is unbounded."),
+        7: (2, "The problem is infeasible."),
+        12: (1, "Iteration limit reached."),
+    }
+    # "Iteration limit reached.",
+    # "The problem appears infeasible, as the phase one auxiliary "
+    # "problem terminated successfully with a residual of {0:.1e}, "
+    # "greater than the tolerance {1} required for the solution to "
+    # "be considered feasible. Consider increasing the tolerance to "
+    # "be greater than {0:.1e}. If this tolerance is unnaceptably "
+    # "large, the problem is likely infeasible.",
+    # "The problem is unbounded, as the simplex algorithm found "
+    # "a basic feasible solution from which there is a direction "
+    # "with negative reduced cost in which all decision variables "
+    # "increase.",
+    # "Numerical difficulties encountered; consider trying "
+    # "method='interior-point'.",
+
+    c, A_ub, b_ub, A_eq, b_eq, bounds, x0 = lp
+
+    lb, ub = bounds.T.copy()                # separate bounds, copy->C-cntgs
+    # highs_wrapper solves LHS <= A*x <= RHS, not equality constraints
+    lhs_ub = -np.ones_like(b_ub)*np.inf     # LHS of UB constraints is -inf
+    rhs_ub = b_ub                           # RHS of UB constraints is b_ub
+    lhs_eq = b_eq                           # Equality constaint is inequality
+    rhs_eq = b_eq                           # constraint with LHS=RHS
+    lhs = np.concatenate((lhs_ub, lhs_eq))
+    rhs = np.concatenate((rhs_ub, rhs_eq))
+
+    if issparse(A_ub) or issparse(A_eq):
+        A = vstack((A_ub, A_eq))
+    else:
+        A = np.vstack((A_ub, A_eq))
+    A = csc_matrix(A)
+
+    options = {
+        'presolve': presolve,
+        'sense': 1,  # minimization
+        'solver': solver,
+        'parallel': parallel,
```

Comment:
Does this option work?

Reply:
Yes and no.  Since openmp is disabled it will always use a single process even if parallel
 Simplex with verbose logging enabled will spit out a warning letting you know that you're not really in parallel:
```
14:33:39 [WARNING] Number of OMP threads available = 0 < 1 = Number of HiGHS threads to be used: Parallel performance will be less than anticipated
```
`parallel` option can safely be removed.